AverageLabScore.py

```diff
@@ -0,0 +1,59 @@
+import seaborn
+from dotenv import load_dotenv
+import pymongo
+import os
+import statistics
+import matplotlib.pyplot as plt
+
+#Loading environment variables
+load_dotenv(dotenv_path="CS125.env")
```

Comment:
if you decide to rename your `CS125.env` file to just `.env`, then just `load_dotenv()` should work without you having to specify a path